langchain/vectorstores/qdrant.py

```diff
@@ -205,22 +206,24 @@ def similarity_search(
             query: Text to look up documents similar to.
             k: Number of Documents to return. Defaults to 4.
             filter: Filter by metadata. Defaults to None.
+            score_threshold: Use to filter out results with a low similarity score. Defaults to None.
 
         Returns:
             List of Documents most similar to the query.
         """
-        results = self.similarity_search_with_score(query, k, filter)
+        results = self.similarity_search_with_score(query, k, filter, score_threshold)
         return list(map(itemgetter(0), results))
 
     def similarity_search_with_score(
-        self, query: str, k: int = 4, filter: Optional[MetadataFilter] = None
+        self, query: str, k: int = 4, filter: Optional[MetadataFilter] = None, score_threshold: Optional[float] = None
```

Comment:
Code should be using `*` operator as delimiter otherwise. Named parameters should not be positionally invoked. 

```suggestion
        self, query: str, k: int = 4, filter: Optional[MetadataFilter] = None, *, score_threshold: Optional[float] = None
```